float64 start_lat
float64 start_long
float64 current_lat
float64 current_long
float64 goal_lat
float64 goal_long
